### PR TITLE
accessibility @ 100%

### PR DESCRIPTION
**Pull request by SaraTabitha**
merged upstream~

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -70,4 +70,60 @@ function foobar_setup() {
...
+* replace removed function with child function that allows zooming
+*/
+function et_add_viewport_meta_2(){
+    echo '<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=2.0, user-scalable=1" />';
+}
+add_action('wp_head', 'et_add_viewport_meta_2');
+?>
+
+<?php
+/*
```
Comment (musicaljoeker) [inline functions.php]: Please comment this better so I understand what it is doing. What does the parameter it take in doing, and what it is returning?

File: functions.php [hunk 2]
```diff
@@ -70,4 +70,60 @@ function foobar_setup() {
...
+
+function wpsites_modify_comment_form_text_area($arg) {
+    $arg['comment_field'] = '<p class="comment-form-comment"><label for="comment">' . _x( 'Comment', 'noun' ) . '</label><textarea id="comment" name="comment" cols="45" rows="8" maxlength="65525" aria-label="comment" aria-required="true"></textarea></p>';
+    return $arg;
+}
+
+add_filter('comment_form_defaults', 'wpsites_modify_comment_form_text_area');
+
+/*
+* altering comment fields so that they also have aria labels
```
Comment (musicaljoeker) [inline functions.php]: Comment this better as well.

File: header.php [hunk 3]
```diff
@@ -56,7 +56,8 @@
 <!--<![endif]-->
 <head>
 	<meta charset="<?php bloginfo( 'charset' ); ?>" />
-	<title><?php wp_title(); ?></title>
+	<meta name="theme-color" content="#fec424"/>
+	<title><?php wp_title(); ?> University of Wisconsin Oshkosh</title>
```
Comment (musicaljoeker) [inline header.php]: Why change the title of the webpage here?

File: searchform.php [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?php
```
Comment (musicaljoeker) [inline searchform.php]: What is this new file for? Please add comments describing its use.

File: style.css [hunk 5]
```diff
@@ -386,9 +386,27 @@ form.gsc-search-box input {
 #footer-column-4 {
   float: left;
 }
+<<<<<<< HEAD
```
Comment (musicaljoeker) [inline style.css]: Need to resolve merge conflicts in this file. [Here is a good Github article](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/) showing how to do this. If you are having problems figuring this out, Jon Ly, Sean, or Tyler should be able to help you - especially Jon.

File: header.php (hunk 3 above)
Comment (SaraTabitha) [reply header.php]: on wptheme for some reason the wp_title comes up blank and the audit doesn't like a blank title element

**Comment by SaraTabitha** [comment]
I made changes o:

File: header.php (hunk 3 above)
Comment (musicaljoeker) [reply header.php]: Might this be because there was no title for the page?

**Pull request merged**